src/commonMain/kotlin/fr/acinq/lightning/serialization/v1/ChannelState.kt

```diff
@@ -270,15 +270,7 @@ data class ChannelVersion(@Serializable(with = ByteVectorKSerializer::class) val
         val channelConfig = fr.acinq.lightning.channel.ChannelConfig.standard
 
         // These are the corresponding channel features
-        val channelFeatures = fr.acinq.lightning.channel.ChannelFeatures(
-            setOf(
-                Feature.Wumbo,
-                Feature.StaticRemoteKey,
-                Feature.AnchorOutputs,
-                Feature.ZeroReserveChannels,
-                Feature.ZeroConfChannels,
-            )
-        )
+        val channelFeatures = fr.acinq.lightning.channel.ChannelFeatures(setOf(Feature.StaticRemoteKey, Feature.AnchorOutputs, Feature.ZeroReserveChannels))
```

Comment:
I think we should leave old channels as-is.

Reply:
Reverted in https://github.com/ACINQ/lightning-kmp/pull/372/commits/d158718b3f49b8e72fe737be4cda2b35c789dcfe